src/os/unix/unix.cpp

```diff
@@ -288,7 +292,20 @@ bool GetClipboardContents(char *buffer, const char *last)
 #endif
 
 
-#ifndef __APPLE__
+#if defined(__EMSCRIPTEN__)
+EM_JS(void, em_open_in_browser, (const char *url, int url_len), {
```

Comment:
Would you mind putting this function in `pre.js`? Would mean we have all Javascript functions in a single place, nice and tidy :)

Reply:
Makes sense; I've moved it.